src/schema/v2/__tests__/startIdentityVerificationMutation.test.ts

```diff
@@ -0,0 +1,90 @@
+import { runQuery, runAuthenticatedQuery } from "schema/v2/test/utils"
+import { StartIdentityVerificationGravityOutput } from "lib/loaders/loaders_with_authentication/gravity"
+
+const mutation = `
+mutation {
+  startIdentityVerification(input: { identityVerificationId: "id-123"}) {
+    startIdentityVerificationResponseOrError {
+      ... on StartIdentityVerificationSuccess {
+        identityVerificationWizardUrl
+        identityVerificationId
+      }
+      ... on StartIdentityVerificationFailure {
+        mutationError {
+          type
+          message
+          detail
+        }
+      }
+    }
+  }
+}
+`
+
+describe("starting an identity verification", () => {
+  it("requires an access token", () => {
+    runQuery(mutation).catch(error => {
```

Comment:
Given that this runs async, you need a way to let jest know when the test ends.  

You can either add `async` before the start of the arrow function on line 25 and an `await` before this `runQuery` statement _or_ you can return `runQuery`.

Reply:
(Could also pass in `done` to it, and then call that after `expect`)

Reply:
I'll update to include `async`, but interesting that this test passed without any further changes needed.

Reply:
Done in f5f9dae34b34a21f8483e1e19992c9ba1cffddf0